spk/python310/src/requirements.txt

```diff
@@ -18,33 +13,49 @@ distlib==0.3.3
 filelock==3.3.2
 Flask==2.0.2
 future==0.18.2
-html5lib==1.1
-idna==3.3
-importlib-metadata==4.8.1
-importlib-resources==5.3.0
-itsdangerous==2.0.1
-Jinja2==3.0.2
-olefile==0.46
-oauthlib==3.1.1
-packaging==21.0
-paramiko==2.8.0
-platformdirs==2.4.0
-pyasn1==0.4.8
-pycparser==2.20
-PyJWT==2.3.0
-pyOpenSSL==21.0.0
-pyparsing==3.0.1
-python-dateutil==2.8.2
-pytz==2021.3
-requests==2.26.0
-requests-oauthlib==1.3.0
-requests-toolbelt==0.9.1
-ruamel.yaml==0.17.16
-six==1.16.0
-tzlocal==4.0.1
-Unidecode==1.3.2
-urllib3==1.26.7
-virtualenv==20.9.0
-webencodings==0.5.1
-Werkzeug==2.0.2
-zipp==3.6.0
+pure:html5lib==1.1
+pure:idna==3.3
+pure:importlib-metadata==4.8.1
+pure:importlib-resources==5.3.0
```

Comment:
Since I'm a python noob I just want to comment on the style, I would prefer 2 separate files rather than prefixing. I know some text editors make is easier but still that's still a lot of text to change. It also departs what is otherwise is a valid requirments.txt that developers are already familiar with.

Reply:
Good points.  This made me thinking, could we do both?

Commit below https://github.com/SynoCommunity/spksrc/pull/4951/commits/c9f1ef0dfe5d769779c93808b2d57749752d223f tries to alleviate this by:
a. allow using distinct files such that if the filename is `requirements-pure.txt` then consider it `pure`.
b. same behaviour applies to `requirements-cross.txt` as `cross` (similar to current)
c. for anything else, consider using `cross`|`pure` prefixes, if no prefix fallback to `pure`

This allows keeping already modified files to stay "as-is" because:
1. `requirements.txt` was used for `pure` with no prefix with recent changes, so still good to go using catch-all c) and sort of as backward compatible as we can get
2. `requirements-cross.txt` behavior stays as it is
3. and it allows a mixture of possibilities to play with to get things to work as you'd like where you can:
* Use the `WHEELS` variable with or direct package naming, with or without prefix
* Use the filename you desire (using prefix for non-default)
* Change default prefix
* etc.

Reply:
@th0ma7 🤩 Thanks 👍